packages/terra-show-hide/src/ShowHide.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Toggle from 'terra-toggle';
+import { injectIntl, intlShape } from 'terra-base';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './ShowHide.module.scss';
+import Button from './_ShowHideButton';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Content in the body of the component that will be shown or hidden.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * The intl object to be injected for translations. Provided by the injectIntl function.
+   */
+  intl: intlShape.isRequired,
+  /**
+   * Callback function triggered when the component is expanded or collapsed.
+   */
+  onChange: PropTypes.func.isRequired,
+  /**
+   * Applies style for which side of the container the button aligns to. Available options are start, center and end
+   */
+  buttonAlign: PropTypes.string,
+  /**
+   * Button text that will be displayed.
+   */
+  buttonText: PropTypes.string,
+  /**
+   * Allows parent to toggle the component. True for open and false for close.
+   */
+  isOpen: PropTypes.bool,
+  /**
+   * Elements(s) that will be visible to the user when component is collapsed
+   */
+  preview: PropTypes.node,
+};
+
+const defaultProps = {
+  buttonAlign: 'start',
+  isOpen: false,
+  preview: <div />,
+};
+
+injectIntl(({ intl }) => {
```

Comment:
Does this throw the error if intl is missing? I figured this wouldn't fire until its rendered, and it never will be rendered.

Reply:
Ah that makes sense why I get a different error message in this test: https://github.com/gabeparra01/terra-core/blob/new_show_hide_component/packages/terra-show-hide/tests/jest/ShowHide.test.jsx#L123

Removed it: https://github.com/cerner/terra-core/pull/1956/commits/7c0c6b4298545df33f98a58cf3236ce4813dcfac